### PR TITLE
Release 0.3.2 

### PR DESCRIPTION
**Pull request by float3**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-## 0.3.2 - January 24th, 2023
+## 0.3.2 - March 12th, 2023
 ### New features
 - Added integration with ytdlp for easy testing of AudioLink shaders in avatar projects. The AudioLinkAvatar prefab now has an UI that lets you paste in a YouTube link, the audio of which will be used to drive AudioLink. (Thanks, rRazgriz)
 - A global shader keyword, "AUDIOLINK_IMPORTED" will now be set automatically when AudioLink is imported.
 ### Changes
 - Add FAQ.md containing frequently asked questions and answers.
 - Add issue templates. Submit issues with them [here](https://github.com/llealloo/vrc-udon-audio-link/issues/new/choose).
+- Improved Docs (Thanks, orels1)
+- Separated repo into 2 Unity projects
```
Comment (pema99) [inline CHANGELOG.md:11]: this is fine but kinda pointless to put in the changelog, since it isn't user facing in any way. the changelog is for users

**Review by pema99 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (float3) [reply CHANGELOG.md:12]: want to remove this? I don't mind

Comment (pema99) [reply CHANGELOG.md:12]: ya

**Comment by float3** [comment]
Reminder to:
- [ ] test on mac
- [ ] add warnings for Linux (and potentially Mac) to the yt-dlp code

**Pull request merged**